### PR TITLE
RCT-102: Using Create React App

### PR DESCRIPTION
**Pull request by rchie01**
# Related Tickets

https://codeandtheory.atlassian.net/browse/RCT-130

# Description

- Creates a simple React Application using `create-react-app`
- This PR intends to demonstrate the use of `create-react-app`

# Validation

- Clone the repository using SSH or HTTPS
- Open the search-app folder located in the RCT-102 folder.
- Run `npm start`

The user should be able to see components imported from `react-bootstrap`.

# Risks
- N/A

### CODE REVIEW

File: RCT-102/search-app/src/components/SearchForm.js [hunk 1]
```diff
@@ -0,0 +1,18 @@
...
+        <Form.Label>Search</Form.Label>
+        <Form.Control type="email" placeholder="Search for something..." />
+      </Form.Group>
+      
+      <Button variant="primary" type="submit">
+        Search
+      </Button>
+    </Form>
+)
+
```
Comment (lycoangelo) [inline RCT-102/search-app/src/components/SearchForm.js]: Please be consistent with the indentation. Let's just use two spaces.

File: RCT-102/search-app/src/components/Results.js
Comment (lycoangelo) [inline RCT-102/search-app/src/components/Results.js:1]: Let's just use two spaces for indentations.

File: RCT-102/search-app/src/App.js [hunk 2]
```diff
@@ -0,0 +1,33 @@
+import { Container, Row, Col, Card } from 'react-bootstrap'
+import SearchForm from './components/SearchForm';
+import Results from './components/Results';
+
+function App() {
+  return (
+    <div >
```
Comment (lycoangelo) [inline RCT-102/search-app/src/App.js]: There's an unnecessary space.

File: RCT-102/search-app/src/App.js [hunk 3]
```diff
@@ -0,0 +1,33 @@
...
+              <SearchForm />
+            </Row>
+            <Row>
+              <Results />
+            </Row>
+          </Col>
+        </Container>
+      </Card.Body>
+    </Card>
+    </div>
```
Comment (lycoangelo) [inline RCT-102/search-app/src/App.js]: Please fix the indentations.

**Review by lycoangelo — CHANGES_REQUESTED** [review]

**Comment by rchie01** [comment]
The aforementioned changes requested above were addressed in the most recent commit. @lycoangelo

**Pull request closed without merging**